### ISSUE TITLE
Not responsive between the sizes of 480px and 600px

### ISSUE THREAD

**Issue by addstar34** [mjmlio/mjml#2178]
**Describe the bug**
Mjml is not responsive between the sizes of 480px and 600px (see screenshots below). Note this is probably a minor issue due to the number of clients that are within this size but I think it might be a fairly easy fix 😬. The issue appears that the compiled mjml sets a width and a max-width, if we just have the max-width it should be fine:

```css
    @media only screen and (min-width:480px) {
      .mj-column-px-600 {
        width: 600px !important;
        max-width: 600px;
      }
    }
```

**To Reproduce**
1. Create a file with this MJML code:

```mjml
<mjml>
  <mj-body background-color="#F0F5F7" font-family="Arial, sans-serif" color="#ffffff" width="100%">

    <mj-section background-color="#00C278" background-repeat="no-repeat" padding="48px 0" text-align="center" vertical-align="top">
      <mj-column width="600px">
        <mj-text align="left" color="#ffffff" font-size="32px" line-height="40px" font-family="Arial, sans-serif" padding="0px 25px">
          <p class="title" style="margin: 0">Main heading</p>
        </mj-text>
        <mj-text css-class="sub-title" align="left" color='#025652' font-size="22px" line-height="30px" font-family="Arial, sans-serif" font-weight="700" padding="8px 25px 0 25px">
          <p class="sub-title" style="margin: 0">Sub heading</p>
        </mj-text>
      </mj-column>
    </mj-section>

    <mj-section background-color="#F0F5F7">
      <mj-column width="600px">
        <mj-table>
          <tr>
            <td>
              <div style="background: url('./aaa-waterfront-1.png') white no-repeat center center;background-size: cover;border-radius:5px;width: 100px;height: 100px;"></div>
            </td>
            <td style="padding-left: 25px">
              <h2 style="font-family: Arial, sans-serif;font-size: 22px;color: #002842;font-weight: 700;margin: 0px; margin-bottom: 14px">Lorem ipsum dolor sit amet</h2>
              <p class='listing-details' style="font-family: Arial, sans-serif;font-size: 18px;font-weight: 400;color: #8aa0ae;margin: 0px">Lorem ipsum dolor sit amet, consectetur.</p>

            </td>
          </tr>
        </mj-table>

      </mj-column>
    </mj-section>


    <mj-section background-color="#ffffff" padding='0 0 0 0'>
      <mj-column width="600px">
        <mj-text font-family="Arial, sans-serif" padding="48px 25px 0px" color="#002842" font-size="20px" line-height="28px">
          <p style="margin: 0px;">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur congue metus et massa malesuada tristique. Aenean lacinia nisi nisi, vel ultricies justo convallis eu. Nullam dictum elementum urna, porta dapibus massa tempus sed. Ut lacinia eget nibh ac tincidunt. Pellentesque aliquet ipsum vel eros condimentum, a suscipit dolor lobortis. Cras vitae odio ac nisi laoreet iaculis vitae in tortor. In eu urna at quam posuere lacinia. Donec blandit sed quam tempor convallis.</p>
        </mj-text>

        <mj-button background-color="#00C278" align="left" color="#ffffff" font-family="Arial, sans-serif" font-size="20px" font-weight="700" href="#" width="260px" padding="30px 25px 0">
          Call to action
        </mj-button>

        <mj-text font-family="Arial, sans-serif" color="#002842" font-size="20px" line-height="28px" padding="30px 25px 48px">
          <p style="margin: 0px;">Lorem ipsum dolor sit amet, consectetur adipiscing elit. In feugiat.</p>
        </mj-text>
      </mj-column>
    </mj-section>


    <mj-section background-color="#F0F5F7">
      <mj-column width="600px">
        <mj-text font-family="Arial, sans-serif" align="center" font-size="14px" line-height="28px" color="#8AA0AE" padding="0 25px">
          <p style="margin: 0">© Lorem ipsum dolor sit.</p>
        </mj-text>

        <mj-text font-family="Arial, sans-serif" align="center" font-size="14px" line-height="28px" color="#8AA0AE" padding="0 25px">
          <p style="margin: 0"><a href="#" style="color: #8AA0AE">email@email.com</a> • <a style="color: #8AA0AE" href='#".'>Unsubscribe</a></p>
        </mj-text>
      </mj-column>
    </mj-section>

  </mj-body>
</mjml>
```
2. Render it to HTML by doing `mjml template.mjml -o template.html`
3. View in browser and shrink browser to a size between 481px and 599px
4. See that the text is being cut off and the footer is not centered properly

**Expected behavior**
Responsive across all sizes.

**MJML environment (please complete the following information):**
 - OS: MacOS
 - MJML Version 4.8.1
 - MJML tool used npm/yarn with nodejs cli

**Affected email clients (for rendering issues):**
Browser (google chrome)

**Screenshots**
In the screenshot notice how the text is not wrapping and the footer text is not centered properly.
<img width="518" alt="image" src="https://user-images.githubusercontent.com/13564150/107895494-48551f80-6f7f-11eb-9e99-25d213599b7d.png">

When I comment out the the width it fixes the issue:
![image](https://user-images.githubusercontent.com/13564150/107895796-493a8100-6f80-11eb-8c12-a24ea474b010.png)

**Comment by iRyusa** [comment]
We can't really play with `max-width` as it fix some German clients (#919). You're still free to add those extra media query with `mj-style` if you want

**Comment by addstar34** [comment]
> We can't really play with `max-width` as it fix some German clients (#919). You're still free to add those extra media query with `mj-style` if you want

Ok no worries, after doing a quick search there doesn't appear to be any devices with resolutions that would fall into 481px and 599px so probably wouldn't impact many users.

Using your suggestion I overwrote the default media query. Here it in case anyone else is concerned with this:

```
<mj-head>
  <mj-style>
    @media only screen and (min-width:480px) {
      .mj-column-px-600 {
        width: auto !important;
        max-width: 600px;
      }
    }
  </mj-style>
</mj-head>
```

Thanks for the quick reply iRyusa.
